source/game/field/obj/ObjectTwistedWay.cc

```diff
@@ -0,0 +1,355 @@
+#include "ObjectTwistedWay.hh"
+
+#include "game/field/CollisionDirector.hh"
+#include "game/system/RaceManager.hh"
+
+#include <egg/math/Math.hh>
+
+namespace Field {
+
+/// @addr{0x80813BD4}
+ObjectTwistedWay::ObjectTwistedWay(const System::MapdataGeoObj &params) : ObjectDrivable(params) {}
+
+/// @addr{0x80814918}
+ObjectTwistedWay::~ObjectTwistedWay() = default;
+
+/// @addr{0x80813c40}
+void ObjectTwistedWay::init() {}
+
+/// @addr{0x80813cfc}
+void ObjectTwistedWay::calc() {
+    if (!System::RaceManager::Instance()->isStageReached(System::RaceManager::Stage::Race)) {
+        m_ac++;
```

Comment:
```suggestion
        ++m_preRaceTimer;
```
Prefer prefix incrementor for consistency with other files.